shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/presentation/StepQuizFeature.kt

```diff
@@ -82,13 +86,23 @@ interface StepQuizFeature {
 
         object ProblemOfDaySolvedModalGoBackClicked : Message
 
+        /**
+         * Daily limit reached modal
+         */
+
+        object ShowProblemsLimitReachedModal : Message
```

Comment:
This message is sent by Dispatcher, but is named like sent message sent from ui. Let's name it like `FreemiumLimitReached` to not define what to don in Dispatcher.

Reply:
I used `ProblemsLimitReached` because it's more concrete